app/views/warehouse_reports/ad_hoc_analysis/_parameters.haml

```diff
@@ -0,0 +1,14 @@
+.report-parameters
+  - options.each do |k,v|
+    - next if v.blank? || v == ['']
+    .report-parameters__parameter
+      %label.label.label-default.parameter-label
+        = k.humanize.downcase
+      %label.label.label-primary.parameter-value
+        %strong
+          - # FIXME: this logic should be moved into WarehouseReports::Export and duplicated into the youth export report
```

Comment:
Did you want to do this now, or is this a FIXME for the future?

Reply:
That was for the future.